chatcommunicate.py

```diff
@@ -147,21 +170,34 @@ def watcher(ev, wrap2):
             if len(message_with_reply) <= 500 or "\n" in reply:
                 ev.message.reply(reply, False)
     else:
-        r = handle_commands(content_source.lower(), message_parts, ev_room, ev_room_name, ev_user_id, ev_user_name, wrap2, content_source, message_id)
-        if type(r) != tuple:
-            r = (True, r)
-        if r[1] is not None:
+        result = handle_commands(
+            content_source.lower(),
+            message_parts,
+            ev_room,
+            ev_room_name,
+            ev_user_id,
+            ev_user_name,
+            wrap2,
+            content_source,
+            message_id
+        )
+        if type(result) != tuple:
+            result = (True, result)
+        if result[1] is not None:
             if wrap2.host + str(message_id) in GlobalVars.listen_to_these_if_edited:
                 GlobalVars.listen_to_these_if_edited.remove(wrap2.host + str(message_id))
-            message_with_reply = u":{} {}".format(message_id, r[1])
-            if len(message_with_reply) <= 500 or "\n" in r[1]:
-                ev.message.reply(r[1], False)
-        if r[0] is False:
+            message_with_reply = u":{} {}".format(message_id, result[1])
+            if len(message_with_reply) <= 500 or "\n" in result[1]:
+                ev.message.reply(result[1], False)
+        if result[0] is False:
             add_to_listen_if_edited(wrap2.host, message_id)
 
 
 def handle_commands(content_lower, message_parts, ev_room, ev_room_name, ev_user_id, ev_user_name, wrap2, content, message_id):
-    message_url = "//chat." + wrap2.host + "/transcript/message/" + str(message_id)
+    message_url = "//chat.{host}/transcript/message/{id}#{id}".format(
+        host=wrap2.host,
+        id=str(message_id)
```

Comment:
`string.format` should be able to do automatic casting of integers to strings, is there any reason why we aren't using that here (and other places)?